src/Surfnet/StepupGateway/GatewayBundle/Resources/views/Gateway/unrecoverableError.html.twig

```diff
@@ -5,5 +5,9 @@
 {% block content %}
     <p class="bg-danger">
         {{ 'gateway.error.unrecoverable_error'|trans }}
+        {% if message is not null %}
+            <p>{{ 'gateway.error.unrecoverable_error_message_intro'|trans }}<br>
```

Comment:
Don't nest paragraph elements. A paragraph may only contain [phrasing content](http://w3c.github.io/html/single-page.html#phrasing-content) and a paragraph itself is not phrasing content.